client/my-sites/sidebar-unified/style.scss

```diff
@@ -34,14 +38,13 @@ $font-size: rem( 14px );
 			font-size: $font-size;
 			font-weight: 400;
 			line-height: 1.3;
-			padding: 0;
+			padding: 0 0 0 8px;
 			color: var( --color-sidebar-text );
 			align-items: center;
-			padding-left: 8px;
 
 			&:hover {
```

Comment:
Do we need a `:focus` here? If not we should comment to justify why it's not required.

Reply:
Good catch, added!